platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/AbstractEmfEntityInputEngine.java

```diff
@@ -0,0 +1,40 @@
+package com.softicar.platform.emf.attribute.field.foreign.entity.input;
+
+import com.softicar.platform.common.core.utils.DevNull;
+import com.softicar.platform.db.runtime.table.IDbTable;
+import com.softicar.platform.dom.elements.input.auto.DomAutoCompleteDefaultInputEngine;
+import com.softicar.platform.emf.entity.IEmfEntity;
+import java.util.Collection;
+
+/**
+ * An input engine for {@link IEmfEntity} instances.
+ *
+ * @author Alexander Schmidt
+ * @author Oliver Richers
+ */
+public abstract class AbstractEmfEntityInputEngine<E extends IEmfEntity<E, ?>> extends DomAutoCompleteDefaultInputEngine<E> {
```

Comment:
Hm, this abstract class isnt used, not even in `EmfEntityInputEngine`
Whats this class for?

Reply:
it is used in EAS a lot...it is the replacement for `AbstractEmfDependentAutoCompleteInputEngine`...without it, the adaptions in EAS would be much bigger, so I thought for now, lets keep this class and maybe remove it in the future